etc/metrics/org.osg.general.perfsonar-simple.conf

```diff
@@ -4,17 +4,22 @@ probe-interval = 60
 job-timeout   = 1500
 no-ping = True
 [org.osg.general.perfsonar-simple args]
-username = perfsonar
-key = ab132837b1f49139e7d1f1f635956ee15c37068e
+username = perfsonar3
+#This is the good key
+key = a4dfdf481a73f37b9dad39528c6150f2944e0934
+# this is the wrong key
+#key = ab132837b1f49139e7d1f1f635956ee15c37068
 goc = http://fermicloud077.fnal.gov:9090
-start = 910
+start = 26052
 timeout = 1400
 debug = False
 summary = True
-maxstart = 86400
+maxstart = 21625672
```

Comment:
Well this is a knob and the rpm does keep whatever it was before. But I will change it and merge. Thanks for looking into it @marian-babik